server/sonar-server/src/main/java/org/sonar/server/component/ws/TreeAction.java

```diff
@@ -182,6 +190,16 @@ private TreeWsResponse doHandle(TreeWsRequest treeWsRequest) {
     }
   }
 
+  private ComponentDto loadComponent(DbSession dbSession, TreeWsRequest request) {
+    String componentId = request.getBaseComponentId();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getBaseComponentId"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)